uk_ua.lang

```diff
@@ -1,154 +1,578 @@
+gui.xaero_radar_render_radar_over_frame=Відображати радар над рамкою
+gui.xaero_radar_over_map_never=Ніколи
+gui.xaero_radar_over_map_list=Список гравців
+gui.xaero_radar_over_map_always=Завжди
+gui.xaero_compass_over_everything=Відображати компас над усім
+gui.xaero_box_compass_over_everything=Відображати літери компаса (Пв, Сх, Пд, Зх) над усіма елементами мапи, а не під деякими, наприклад маркери.
+gui.xaero_waypoint_server_shared=Сервер
+gui.xaero_waypoint_somebody_shared=Хтось
+gui.xaero_pac_players=Відобразити група/союзники
+gui.xaero_box_pac_players=Відображати гравців, місцезнаходження яких вам відоме (за допомогою мода "Open Parties and Claims"), коли вони виходять за межі рамки мінімапи або їх не видно з іншої причини.
+gui.xaero_toggle_pac_players=Перемикання група/союзники
+gui.xaero_toggle_pac_chunk_claims=Перемикнути відображення претензій до чанків
+gui.xaero_pac_marked_for_forceload= (Прискорене завантаження)
+gui.xaero_pac_claim_tooltip=%1$s Заява%2$s
+gui.xaero_pac_server_claim_tooltip=Заява сервера%1$s
+gui.xaero_pac_expired_claim_tooltip=Заява сплинула%1$s
+gui.xaero_pac_claims=Відбивання претензій до чанків
+gui.xaero_box_pac_claims=Відображення претензій до чанків з моду Open Parties and Claims у вигляді кольорового накладання.
+gui.xaero_pac_claims_border_opacity=Непрозорість межі претензій до чанків.
+gui.xaero_box_pac_claims_border_opacity=Рівень прозорості межі кольорового накладання претензій до чанків. Для відображення претензій до чанків необхідно встановити мод Open Parties and Claims.
+gui.xaero_pac_claims_fill_opacity=Непрозорість заповнення претензій до чанків.
+gui.xaero_box_pac_claims_fill_opacity=Рівень прозорості кольорового заповнення претензій до чанків. Для відображення претензій до чанків необхідно встановити мод Open Parties and Claims.
+gui.xaero_pac_current_claim=Відображати поточне володіння
+gui.xaero_box_pac_current_claim=Відображати інформацію про володіння у чанку, в якому ви перебуваєте. Володіння чанками потребує Open Parties and Claims мод.
+gui.xaero_category_include_list_include_in_super=Включити до супер-категорії
+gui.xaero_box_category_include_list_include_in_super=Автоматично включати всі сутності з цього списку та інших списків, включених до цієї категорії, до суперкатегорії/вищої категорії, окрім сутностей зі списку винятків. Таким чином, вам не доведеться вручну додавати дублікати на кожному рівні.
+gui.xaero_temp_waypoints_global=Тимчасові глобальні маркери
+gui.xaero_box_temp_waypoints_global=Коли увімкнено, тимчасові маркери розглядаються як глобальні та ігнорують обмеження на їх відображення.
+gui.xaero_keep_enlarged_minimap_unlocked=Зберігати напрямок при збільшення
```

Comment:
Does not seem correct. Should be "keep rotation" not "keep direction". The minimap will continue to rotate, not keep a specific angle.